mypy/stubtest.py

```diff
@@ -825,6 +825,13 @@ def verify_funcitem(
         if not callable(runtime):
             return
 
+    if isinstance(stub, nodes.FuncDef):
+        stub_abstract = stub.abstract_status == nodes.IS_ABSTRACT
```

Comment:
I think we can safely skip `nodes.IMPLICITLY_ABSTRACT` here. Because it can only happen for a protocol.